ci/ray_ci/tester.py

```diff
@@ -71,11 +72,11 @@ def main(
     if not bazel_workspace_dir:
         raise Exception("Please use `bazelisk run //ci/ray_ci`")
     os.chdir(bazel_workspace_dir)
+    docker_login(_DOCKER_ECR_REPO.split("/")[0])
```

Comment:
this docker_login not requires since we pull the docker image from the internet; previously, it just uses the local cache